src/background/services/background.ts

```diff
@@ -167,6 +167,7 @@ export class Background {
       if (details.reason === 'install') {
         await this.storage.populate()
         await this.openPaymentsService.genererateKeys()
+        await this.storage.populateOverpayingSessions()
```

Comment:
Instead of saving them in the extension storage, shouldn't we save them in a state service in the background?

Reply:
Just in case, might not work out to store it in memory, as background service worker is killed after some inactivity.
But `storage.session` may work well enough?

Reply:
Agree on not keeping it in primary extension storage though.
Session storage has added benefit of not needing to do https://github.com/interledger/web-monetization-extension/issues/287 maybe?

Reply:
Can we specify a TTL for the items that are added in the session storage?

Reply:
We can't, but we can have a guarantee that it goes away when browser is closed.
To save storage, we'd need to clear expired items ourselves (which we can do while setting a value, instead of requiring a new worker service).

Reply:
Just checked `storage.session` is available from:
- Chrome 102 (May 26, 2022)
- Safari 16.4 (March 27, 2023)
- Firefox 115 (July 4, 2023)

So, might need to use `storage.local` only for now.